assets/src/data/persistence/resource.ts

```diff
@@ -20,6 +20,18 @@ export function edit(
 ) {
   const update = Object.assign({}, pendingUpdate, { releaseLock });
 
+  // Index "citation references" in the "content" and elevate them as top-level list
+  const citationRefs: BibPointer[] = [];
+  traverseContent(update.content.model, (key: string, value: any) => {
+    if (!value) {
+      return;
+    }
+    if (value.type === 'cite') {
+      citationRefs.push({ type: 'bibentry', id: value.bibref });
```

Comment:
Now that activities are removed from this collection, we can change the shape and simply store the ids as opposed to an object (since there is only one type of object)

Reply:
I made the tweaks we talked about in the function assemble_bib_entries and refactored its location so it makes sense for use in both authoring and delivery. 

Simplified the bibrefs from object array to simple string array